### PR TITLE
fixed static file print in detailed dry-run

### PR DESCRIPTION
**Pull request by roironn**

### CODE REVIEW

File: packages/cli/test/formatter.test.ts [hunk 1]
```diff
@@ -269,6 +269,25 @@ describe('formatter', () => {
           expect(output).toMatch(new RegExp(`${objectType.elemID.name}.*annotations.*label.*TYPE: string`, 's'))
         })
       })
+      describe('with static file value', () => {
+        const staticFileBefore = new StaticFile({
+          content: Buffer.from('FAFAFAFAFAFAFAFAFAFA'),
```
Comment (omrilit) [inline packages/cli/test/formatter.test.ts:274]: I counted it and it sounds like 'Psycho Killer', correct?

**Review by omrilit — APPROVED** [review]

File: packages/cli/test/formatter.test.ts (hunk 1 above)
Comment (roironn) [reply packages/cli/test/formatter.test.ts:274]: LOOK AT THE AFTER ELEMENT TEXT

Comment (omrilit) [reply packages/cli/test/formatter.test.ts:274]: far better? You lost me

Comment (omrilit) [reply packages/cli/test/formatter.test.ts:274]: Had to go to the lyrics to catch that one :)

**Pull request merged**